db/migrations/6.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE results;
+CREATE TABLE results (
+  id SERIAL,
+  driver_id INTEGER,
+  race_id INTEGER,
+  team_id INTEGER,
+  year TEXT,
+  round TEXT,
```

Comment:
If we need to do an order by might we not want them as numbers? If you order strings won't you get 1, 10, 2, 3?